Jenkinsfile

```diff
@@ -92,7 +92,7 @@ def getFullBuild(jdk, os) {
                       globalMavenSettingsConfig: settingsName,
                       //options: [invokerPublisher(disabled: false)],
                       mavenLocalRepo: localRepo) {
-                sh "mvn -V -B install -Dmaven.test.failure.ignore=true -e -Pmongodb -T3 -DmavenHome=${mvntoolInvoker}"
+                sh "mvn -V -B install -Dmaven.test.failure.ignore=true -e -Pmongodb -T3 -DmavenHome=${mvntoolInvoker} -Dunix.socket.tmp="+env.JENKINS_HOME
```

Comment:
Why are we polluting the JENKINS_HOME with temp files now?

Reply:
jnr cannot handle paths such /home/jenkins/workspace/jetty.project_PR-2635-K7E6LNJADYW4DDQGANLMEFBLXHJV7NNIBFPIJX6OUPTQCUYJFXNA/jetty-unixsocket/target/unix13242728516864007402.sock 
JENKINS_HOME on docker slave not master so it's not very long.
well we can use /tmp and totally skip @Before on windows?